webservices/legal_docs/load_legal_docs.py

```diff
@@ -187,8 +199,8 @@ def index_advisory_opinions():
                    "url": pdf_url,
                    "requestor_names": requestor_names,
                    "requestor_types": list(requestor_types),
-                   "citations": sorted(list(citations[(row[8], row[3])])),
-                   "cited_by": sorted(list(cited_by[row[8]])) if row[8] in cited_by else []}
+                   "citations": citations[(row[8], row[3])],
```

Comment:
yes no problem, I should update by standup tomorrow (will need to refactor a bunch of tests)